.gitignore

```diff
@@ -0,0 +1 @@
+small-talk/env.local
```

Comment:
Since this is causing a merge conflict, the current gitignore includes .env.local so we should just accept the existing one as-is.